cmd/security-agent/app/app.go

```diff
@@ -92,6 +94,8 @@ Datadog Security Agent takes care of running compliance and security checks.`,
 func init() {
 	// attach the command to the root
 	SecurityAgentCmd.AddCommand(startCmd)
+	startCmd.Flags().StringVarP(&pidfilePath, "pidfile", "p", "", "path to the pidfile")
```

Comment:
Might want to move this line higher for readability

Reply:
maybe if we reorder the `AddComment` like this 
it can be clearer

```golang
        SecurityAgentCmd.AddCommand(versionCmd)
	SecurityAgentCmd.AddCommand(complianceCmd)

        SecurityAgentCmd.AddCommand(startCmd)
	startCmd.Flags().StringVarP(&pidfilePath, "pidfile", "p", "", "path to the pidfile")
```

Reply:
Done